js/private/js_info.bzl

```diff
@@ -0,0 +1,13 @@
+"JsInfo provider"
+
+JsInfo = provider(
+    doc = "Encapsulates information provided by rules in rules_js and derivative rule sets",
```

Comment:
is there an easy way to summarize what type of "information" or would that basically just be listing the params (source, declarations, npm links)?

Reply:
Borrowed this text from PyInfo: https://docs.bazel.build/versions/main/skylark/lib/PyInfo.html#modules.PyInfo

Generic enough that it will never be out of date :)